index.html

```diff
@@ -13,7 +13,65 @@
   <![endif]-->
 </head>
 
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
-</body>
+ <body>
+      <!-- Copy and paste your HTML from the first UI project here -->
+      <header>
+         <nav class="nav">
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+         </nav>
+         <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </header>
+      <div class="get-started">
+         <section>
+            <h1>
+               Innovation<br>
+               On<br>
+               Demand
+            </h1>
+            <a href="#">Get Started</a>
+         </section>
+         <img id="get-started-code" src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+      <div class="main">
+         <section id="features">
+            <h2>Features</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+         </section>
+         <section id="about">
+            <h2>About</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+         </section>
+         <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+         <section id="services">
+            <h2>Services</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+         </section>
+         <section id="product">
+            <h2>Product</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+         </section>
+         <section id="vision">
+            <h2>Vision</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+         </section>
+      </div>
+      <section id="contact">
+         <h2>Contact</h2>
+         <address>
+            123 Way 456 Street<br>
+            Somewhere, USA<br><br>
+            1 (888) 888-8888<br><br>
+            sales@greatidea.io
+         </address>
+      </section>
+      <footer>
+         <p>Copyright Great Idea! 2018
+         <p>
+      </footer>
+   </body>
```

Comment:
```suggestion
      <div class="main">
         <section id="features">
            <h2>Features</h2>
            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </section>
         
         <section id="about">
            <h2>About</h2>
            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </section>
         
         <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
         
         <section id="services">
            <h2>Services</h2>
            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </section>
         
         <section id="product">
            <h2>Product</h2>
            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </section>
         
         <section id="vision">
            <h2>Vision</h2>
            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
         </section>
         
      </div>
      
      <section id="contact">
         <h2>Contact</h2>
         
         <address>
            123 Way 456 Street<br>
            Somewhere, USA<br><br>
            1 (888) 888-8888<br><br>
            sales@greatidea.io
         </address>
         
      </section>
      
      <footer>
         <p>Copyright Great Idea! 2018
         <p>
      </footer>
   </body>
```

Reply:
Your code looks really clean and somewhat easy to read but i would suggest adding white space like this to make it easier to read your HTML